### PR TITLE
Remove pip installation steps

### PR DESCRIPTION
**Pull request by selvakumaar5496**
Signed-off-by: Selvakumar <selva.k.nambi@seagate.com>

# Problem Statement
- Problem statement 

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

-----
[View rendered py-utils/README.md](https://github.com/selvakumaar5496/cortx-utils/blob/readme/py-utils/README.md)

### CODE REVIEW

File: py-utils/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ A common utils framework which includes common modules across components
 
 ## Prerequisites for build
 ```bash
-sudo yum install gcc rpm-build python36 python36-pip python36-devel python36-setuptools openssl-devel libffi-devel
+sudo yum install gcc rpm-build python36 python36-pip python36-devel python36-setuptools openssl-devel libffi-devel python36-dbus
```
Comment (sachinpunadikar) [inline py-utils/README.md:26]: give exact version

File: py-utils/README.md [hunk 2]
```diff
@@ -33,15 +33,6 @@ git clone --recursive https://github.com/Seagate/cortx-utils -b main
...
-
-  - Create pip package
-    - It will create `cortx_py_utils-1.0.0-py3-none-any.whl`
-```bash
-$ pip3 install wheel
-$ cd ./cortx-utils/py-utils
-$ python3 setup.py bdist_wheel
-```
 
   - Create RPM Package
```
Comment (sachinpunadikar) [inline py-utils/README.md:37]: Remove this line

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me.

File: py-utils/README.md [hunk 3]
```diff
@@ -51,13 +42,8 @@ Run below command from repo root (cortx-utils).
...
 ```
 
-## Installation
-  - Installation with pip package
-```bash
-$ cd ./py-utils/dist
-$ pip3 install cortx_py_utils-1.0.0-py3-none-any.whl
-```
-
+## Install
```
Comment (sachinpunadikar) [inline py-utils/README.md:45]: Installation

File: py-utils/README.md [hunk 4]
```diff
@@ -51,13 +42,8 @@ Run below command from repo root (cortx-utils).
...
+ 
   - Installation with RPM package
```
Comment (sachinpunadikar) [inline py-utils/README.md:47]: Remove this line

File: py-utils/README.md [hunk 5]
```diff
@@ -68,10 +54,6 @@ $ sudo yum install -y cortx-py-utils-*.noarch.rpm
 
 ## Uninstall
 
-- Pip package uninstall
-```bash
-$ pip3 uninstall cortx-py-utils
-```
 
   - RPM uninstall
```
Comment (sachinpunadikar) [inline py-utils/README.md:58]: Remove this line

File: py-utils/README.md [hunk 6]
```diff
@@ -68,10 +54,6 @@ $ sudo yum install -y cortx-py-utils-*.noarch.rpm
 
 ## Uninstall
```
Comment (sachinpunadikar) [inline py-utils/README.md:55]: Un-installation ?

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Work on the comments

File: py-utils/README.md (hunk 2 above)
Comment (selvakumaar5496) [reply py-utils/README.md:37]: done

File: py-utils/README.md (hunk 3 above)
Comment (selvakumaar5496) [reply py-utils/README.md:45]: done

File: py-utils/README.md (hunk 4 above)
Comment (selvakumaar5496) [reply py-utils/README.md:47]: done

File: py-utils/README.md (hunk 6 above)
Comment (selvakumaar5496) [reply py-utils/README.md:55]: done

File: py-utils/README.md (hunk 5 above)
Comment (selvakumaar5496) [reply py-utils/README.md:58]: done

File: py-utils/README.md (hunk 1 above)
Comment (rohit-k-dwivedi) [reply py-utils/README.md:26]: also adding a -y option should give a smoother experiece

Comment (selvakumaar5496) [reply py-utils/README.md:26]: It gave me an error if i provide version.
This above itself resolves the dbus dependency

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**